Composer/packages/server/src/utility/creation.ts

```diff
@@ -4,21 +4,36 @@
 import { BotTemplate, QnABotTemplateId } from '@bfc/shared';
 import union from 'lodash/union';
 
-export const templateSortOrder = [
+import AssetService from '../services/asset';
+
+export const backUpTemplateSortOrder = [
   { generatorName: '@microsoft/generator-bot-empty', displayName: 'Blank bot' },
-  { generatorName: '@microsoft/generator-bot-conversational-core', displayName: 'Basic conversational bot' },
+  { generatorName: '@microsoft/generator-bot-conversational-core', displayName: 'Basic bot' },
   { generatorName: '@microsoft/generator-bot-assistant-core', displayName: 'Basic assistant' },
   { generatorName: '@microsoft/generator-bot-enterprise-assistant', displayName: 'Enterprise assistant' },
   { generatorName: '@microsoft/generator-bot-people', displayName: 'People' },
   { generatorName: '@microsoft/generator-bot-calendar', displayName: 'Calendar' },
   { generatorName: QnABotTemplateId, displayName: 'QnAMaker bot' },
 ];
 
-export const sortTemplates = (templates: BotTemplate[]): BotTemplate[] => {
+export const sortTemplates = async (templates: BotTemplate[]): Promise<BotTemplate[]> => {
   let sortedTemplateList: BotTemplate[] = [];
+  let sortOrder = backUpTemplateSortOrder;
+  try {
+    const templateSortOrder = await AssetService.manager.getRawGithubFileContent(
+      'microsoft',
+      'BotFramework-Composer',
+      'main',
+      'templates.json'
+    );
+    const templateSortOrderObj = JSON.parse(templateSortOrder);
+    sortOrder =
+      templateSortOrderObj && templateSortOrderObj?.length > 0 ? templateSortOrderObj : backUpTemplateSortOrder;
+  } catch {
+    sortOrder = backUpTemplateSortOrder;
```

Comment:
no need

Reply:
instead log an error so we know something is wrong in either fetching or parsing that json

Reply:
Handled